sql-plugin/src/main/scala/com/nvidia/spark/rapids/basicPhysicalOperators.scala

```diff
@@ -210,6 +210,99 @@ trait GpuProjectExecLike extends ShimUnaryExecNode with GpuExec {
   override def outputBatching: CoalesceGoal = GpuExec.outputBatching(child)
 }
 
+abstract class AbstractProjectSplitIterator(iter: Iterator[ColumnarBatch],
```

Comment:
Would be nice to have a Scaladoc comment for this class explaining what it does.